packages/terra-application-links/src/tabs/ApplicationTabs.module.scss

```diff
@@ -6,8 +6,8 @@
     height: 100%;
     justify-content: center;
     overflow: hidden;
-    padding-left: var(--terra-application-tabs-container-side-padding, 1.4285714286rem);
-    padding-right: var(--terra-application-tabs-container-padding-right, 1.4285714286rem);
+    padding-left: var(--terra-application-links-tabs-container-padding-left, 1.4285714286rem);
+    padding-right: var(--terra-application-links-tabs-container-padding-right, 1.4285714286rem);
```

Comment:
Does the linter complain if we remove `tabs` from custom properties?
```scss
var(--terra-application-links-container-padding-left
var(--terra-application-links-container-padding-right)
...
```

Reply:
It does not complain if we remove `tabs`.

Reply:
To me, `links-tabs` is redundant. Anyone else have thoughts? The css class is `application-tabs`, but that's probably remnants of previous design - @dkasper-was-taken probably has insight on this

Reply:
Ditto, current state was weird since package was app-links but css and all files used name of tabs. My 2 cents are there is no reason to keep either, since they are actually buttons (except for the menu which is just a clickable div). 

Could do something agnostic like "--terra-application-links-nav-container-[...]", "--terra-application-links-nav-item-[...]", "--terra-application-links-nav-item-inner-[...]" etc...  if you do variables now, and then save refactor of actual class names (and potentially filenames) for later which would be a lot.

Reply:
Talked at scrum, decided to leave as is.